cli/config/parser.go

```diff
@@ -40,60 +53,39 @@ func ParsePxfCluster(clusterName string) (*PxfCluster, error) {
 	return pxfDeployment.Clusters[clusterName], nil
 }
 
-func readFile(path string) ([]string, error) {
-	data, err := os.ReadFile(path)
+// createCluster creates a PxfCluster given the cluster name and file contents of the cluster file.
+func createCluster(clusterName string) (*PxfCluster, error) {
+	clusterPath := GetPathForClusterFile(clusterName)
+	fileContents, err := FileProvider(clusterPath)
```

Comment:
```suggestion
	fileContents, err := ReadFileAndClose(clusterPath)
```
this name would be more intuitive to read, it indicates an action rather than a noun